src/main/java/com/gluonhq/richtext/model/TextDecoration.java

```diff
@@ -45,6 +48,42 @@ public static Builder builder() {
         return new Builder();
     }
 
+    /**
+     * Returns a new instance of the current TextDecoration
+     * and normalizes properties based on the supplied decoration.
+     * FontWeight and FontPosture normalize into
+     * {@link FontWeight#NORMAL} and {@link FontPosture#REGULAR} respectively.
+     * @param decoration Decoration to compare and normalize
+     * @return New TextDecoration instance with properties normalized
+     */
+    public TextDecoration normalize(TextDecoration decoration) {
```

Comment:
I understand we have a Builder in place, but adding a `toBuilder()` method and two checks, and then build, seems too much work.

Isn't this more simple and straightforward?

```
public TextDecoration normalize(TextDecoration decoration) {
        if (decoration == null) {
            return this;
        }
        TextDecoration d = new TextDecoration();
        d.foreground  = foreground;
        d.background  = background;
        d.fontFamily  = fontFamily;
        d.fontSize    = fontSize;
        d.fontWeight  = fontWeight == decoration.getFontWeight() ? FontWeight.NORMAL : fontWeight;
        d.fontPosture = fontPosture == decoration.getFontPosture() ? FontPosture.REGULAR : fontPosture;
        return d;
    }
```

Reply:
We can even set default posture and weight to simplify this

Reply:
I like this :)

Reply:
@eugener Do you mind explaining what you mean?

Reply:
I'm sorry, my fault - it is already set! :)